functions

```diff
@@ -44,15 +44,15 @@ function add_copy_sym_link_file()
 # @param error message
 function log_error()
 {
-   echo "$($TIME_STAMP) ERROR: $@"
+   echo "${TIME_STAMP} ERROR: $@"
```

Comment:
Where is this variable defined?
Originally, it is a `date` call:
https://github.com/openbmc/phosphor-debug-collector/blob/master/tools/dreport.d/dreport#L51

Reply:
It is defined exactly in the place that you specified. ;)

Reply:
Really? How?
Even so, now instead of timestamp it will print:
`date -t ERROR: blah-blah-blah...`

Reply:
No, I'm wrong. For the CLI it defined [there](https://github.com/YADRO-KNS/obmc-yadro-cli/blob/cc2cd3f387b292392949ba9c36c5c99935fe5234/commands/health.vegman#L62).

Reply:
And you are right, the root of the issue in other place.
I've rework this PR.